stream-chat-android-sample/src/release/kotlin/io/getstream/chat/sample/application/AppConfig.kt

```diff
@@ -0,0 +1,36 @@
+package io.getstream.chat.sample.application
+
+import io.getstream.chat.sample.data.user.SampleUser
+
+object AppConfig {
+    const val apiKey: String = "qk4nn7rpcn75"
+    const val apiUrl: String = "chat-us-east-staging.stream-io-api.com"
+    const val apiTimeout: Int = 6000
+    const val cndTimeout: Int = 30000
+
+    private val user1 = SampleUser(
+        id = "80c26629-bc25-4ee5-a8ae-4824f8097b53",
+        name = "paul",
+        token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoiODBjMjY2MjktYmMyNS00ZWU1LWE4YWUtNDgyNGY4MDk3YjUzIn0.ca55fO4jAbexSSZKqBMT0OaxCZaPtYmo1HWgvjBnOY4",
+        image = "https://getstream.io/random_png?id=80c26629-bc25-4ee5-a8ae-4824f8097b53&name=paul"
+    )
+    private val user2 = SampleUser(
+        id = "a97a21dd-d993-42e0-8e52-31d6b3cea82c",
+        name = "chani",
+        token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoiYTk3YTIxZGQtZDk5My00MmUwLThlNTItMzFkNmIzY2VhODJjIn0.7Hv38I8xq328_nyKdMA8m1ehz3jrNdnaH1GzSj6yuzk",
+        image = "https://getstream.io/random_png?id=a97a21dd-d993-42e0-8e52-31d6b3cea82c&name=chani"
```

Comment:
There is a `size` param to get a smaller image, but seems that it is not working now
(https://getstream.io/random_png/?name=jaap%20bakker&size=200).